pom.xml

```diff
@@ -188,7 +188,7 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>42.5.4</version>
+            <version>42.6.0</version>
```

Comment:
Denne kan man la spring boot velge versjon for. Hvis det ikke er en spesiell grunn til å bruke seneste driver